TheForceEngine/TFE_DarkForces/GameUI/pda.cpp

```diff
@@ -350,6 +352,32 @@ namespace TFE_DarkForces
 		// Ensure that mouse movements are not lost between updates.
 		pda_handleMouseAccum(&wdx, &wdy);
 
+		// Handle Mouse Panning
+		if (TFE_Input::mouseDown(MBUTTON_LEFT))
+		{
+			s32 mouseX, mouseZ;
+			SDL_GetMouseState(&mouseX, &mouseZ);
```

Comment:
TFE_Input has a function to get the mouse position, so you don't need to include SDL here.

Reply:
This is implemented.